src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -118,7 +118,14 @@ sealed trait Task extends Instance {
 }
 
 object Task {
-  /**
+
+  def from(i: Instance): Option[Task] = from(Some(i))
```

Comment:
`from` seemed more clear to me. If `apply` is more idiomatic that's fine

On Thu, Sep 8, 2016 at 8:35 PM, Jason Gilanfarr <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/core/task/Task.scala
> <https://github.com/mesosphere/marathon/pull/4336#discussion_r78113348>:
>
> > @@ -118,7 +118,14 @@ sealed trait Task extends Instance {
> >  }
> >
> >  object Task {
> > -  /**
> > +
> > +  def from(i: Instance): Option[Task] = from(Some(i))
>
> Why not apply?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4336/files/1dbfa88c79764d63f7340e618c5af92001554705#r78113348>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLBI-5BgHzGY2j9lukqkg7atoW32xks5qoKnbgaJpZM4J4k2Q>
> .
>